OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -118,11 +118,17 @@ public SlotDropDownOption(string title, string order, Func<bool> selected)
 				flag.GetImageCollection = () => "flags";
 				flag.GetImageName = () => factionId;
 
-				var factionName = faction.Description.SubstringBefore("\\n", StringComparison.Ordinal);
```

Comment:
Note that `SubstringBefore` and `SubstringAfter` actually come from Stylecop, so I am removing these here to simplify the fix for #13567.